docs/notes/02-packaging.md

```diff
@@ -48,6 +49,106 @@ testing notes](05-linters-and-tests.md) for more info.
 
 TODO: Also need to have guidance on [build tools](https://www.pyopensci.org/python-package-guide/package-structure-code/python-package-build-tools.html)
```

Comment:
this should be removed now, since we have guidance

Reply:
good point. I still think that a suggest change would make it quicker for stuff like that